src/reference/asciidoc/endpoint-summary.adoc

```diff
@@ -66,10 +66,10 @@ To recap, *Inbound Channel Adapters* are used for one-way integration bringing d
 | *Events*
 
 
-| <<applicationevent-inbound>>
+| <<appevent-inbound>>
```

Comment:
I thought it was the length of the link that was causing the problem but it was the length of `ApplicationEvents` in the link's title. I added a space there which fixed it, but I didn't bother with going back to the old link names.